bologna.json

```diff
@@ -0,0 +1,3 @@
+{"pageid": 10987372, "text": "The Johns Hopkins University SAIS Europe in Bologna, Italy, is the European campus of the Paul H. Nitze School of Advanced International Studies (SAIS), a division of Johns Hopkins University located in Washington, D.C.. SAIS Europe offers an interdisciplinary academic program that emphasizes international economics, international relations, languages, and specializations either in functional areas or regional studies. SAIS Europe was founded in 1955 by C. Grove Haines.", "title": "The Johns Hopkins University SAIS Europe", "ref_qid": "Q7743414", "ref_pageid": 10987372, "ref_title": "The Johns Hopkins University SAIS Europe", "ref_point": "(44.4948, 11.3557)", "ref_instance": "university"}
```

Comment:
I'd prefer not to have this file in the top level directory. Can you create cabby/data/example and put it there?

Reply:
Done